### PR TITLE
Qubes summit 2022

### PR DESCRIPTION
**Pull request by Asiderr**
Signed-off-by: Norbert Kamiński <norbert.kaminski@3mdeb.com>

### CODE REVIEW

File: blog/content/post/2022-09-21-qubes-os-summit-2022.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+  - OS Dev
+  - App Dev
+  - Security
+  - Manufacturing
+
+---
+
+## Long story short
+
+Three weeks ago 3mdeb with Qubes OS team had organized next edition of the
```
Comment (pietrushnic) [inline blog/content/post/2022-09-21-qubes-os-summit-2022.md:29]: We organized it with Invisible Things Lab and order should be switched.

File: blog/content/post/2022-09-21-qubes-os-summit-2022.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+Qubes OS summit. The last years [2020](https://blog.3mdeb.com/2020/2020-06-17-qubes_summation/)
+and [2021](#) editions were hosted
+remotely and this year event was a face-to-face event hosted in Berlin, which
+took place from the 9th to the 11th of September. It has been another productive
+event, during which we have exchanged loads of knowledge and expertise.
+
+## Day 1
+
+The first day started with
```
Comment (pietrushnic) [inline blog/content/post/2022-09-21-qubes-os-summit-2022.md:38]: I guess embedded yt video would be better. Description of talks could be more robust adding some personal review or some personal conclusion based on the talk.

File: blog/content/post/2022-09-21-qubes-os-summit-2022.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+  - IoT
+  - Miscellaneous
+  - OS Dev
+  - App Dev
+  - Security
+  - Manufacturing
+
+---
+
+## Long story short
```
Comment (pietrushnic) [inline blog/content/post/2022-09-21-qubes-os-summit-2022.md:27]: hackers say tl;dr

File: blog/content/post/2022-09-21-qubes-os-summit-2022.md [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+to present the most significant differences that an end user user will see when
+installing Dasharo over the original proprietary firmware.
+
+In this presentation I will present actual difference in performance as measured
+by a set of benchmarks, talk about what causes them and what challenges there
+are to bring performance to a state that is competitive with the vendor BIOS.
+We'll also discuss how these differences may impact QubesOS users.
+
+
+## Hackaton
```
Comment (pietrushnic) [inline blog/content/post/2022-09-21-qubes-os-summit-2022.md:261]:
```suggestion
## Hackathon
```
?

Ideally would be if some picture would be added.Of course respecting privacy.

File: blog/content/post/2022-09-21-qubes-os-summit-2022.md (hunk 1 above)
Comment (Asiderr) [reply blog/content/post/2022-09-21-qubes-os-summit-2022.md:29]: Fixed.

File: blog/content/post/2022-09-21-qubes-os-summit-2022.md (hunk 2 above)
Comment (Asiderr) [reply blog/content/post/2022-09-21-qubes-os-summit-2022.md:38]: I added links to the video and I've changed a lot a descriptions of the presentations.

File: blog/content/post/2022-09-21-qubes-os-summit-2022.md (hunk 3 above)
Comment (Asiderr) [reply blog/content/post/2022-09-21-qubes-os-summit-2022.md:27]: I've changed that to the blog title.

File: blog/content/post/2022-09-21-qubes-os-summit-2022.md (hunk 4 above)
Comment (Asiderr) [reply blog/content/post/2022-09-21-qubes-os-summit-2022.md:261]: Fixed here and in a few more places, I've added link to the yt highlights from the hackathon.

**Review by pietrushnic — APPROVED** [review]

**Pull request merged**